BrainPortal/app/views/access_profiles/show.html.erb

```diff
@@ -53,7 +53,7 @@
       <%= render :partial => 'shared/color_picker', :locals => { :step => 30, :greys => false, :dark => false } %>
     <% end %>
 
-    <% t.edit_cell(:description, :content => overlay_description(@access_profile.description), :show_width => 2 ) do %>
+    <% t.edit_cell(:description, :content => @access_profile.description ) do %>
```

Comment:
The formatting that was provided in the overlay from the helper. E.g. if I entered in teh description box

```
This is my description.

It has two three lines, with the middle one a blank line.
```

what I see now is

```
This is my description. It has two three lines, with the middle one a blank line.
```